webClient/src/SamplePage.tsx

```diff
@@ -49,17 +49,28 @@ class SamplePage extends React.Component<any, any> {
                   {/* Action types are used to determine what sort of Action is being taken on whatever App instance is the target. Launch simply creates a new instance with the context you provide, but Message can be used to communicate with an already open Instance to drive some action */}        
                   <label style={styles.boldText}>{t('action_type')}</label>
                   <input type="radio" value="Launch" name="actionType" checked={this.props.actionType == "Launch"} onChange={this.props.handleActionTypeChange}/>
-                  <label htmlFor="actionLaunch">{t('launch')}</label>
+                  <label  style={styles.labelPadding} htmlFor="actionLaunch">Launch</label>
```

Comment:
You must keep t('launch') and related use of t() because it is used for internationalization where the framework finds the translation for the key "launch" and does a substitution.